### PR TITLE
Fixing some cosmetic details on Google Sign In Button

### PR DESCRIPTION
**Pull request by maguero**
### Description
Fixing some cosmetic details on Google Sign In Button to align with Google designs.

### Screenshots

| n-conversion-forms | next-subscribe |
| ------ | ----- |
| image | Screenshot 2021-05-06 at 16 53 27 |

### CODE REVIEW

File: styles/google-sign-in.scss [hunk 1]
```diff
@@ -1,22 +1,23 @@
 @mixin ncfGoogleSignIn() {
 	.google_button {
-		background-color: #4285f4;
+		background-color: #1A73E8;
 		color: #ffffff;
 		display: inline-block;
-		width: 100%;
+		width: 100% !important;
```
Comment (angelaFT) [inline styles/google-sign-in.scss:6]: what's the accessibility best practice on `!important` these days? genuine question.

**Review by angelaFT — APPROVED** [review]

File: styles/google-sign-in.scss (hunk 1 above)
Comment (ekocian) [inline styles/google-sign-in.scss:6]: nitpick: isn't there any other way to do this? !important are quite dangerous, but if this is isolate I think is not a problem

**Review by ekocian — APPROVED** [review]

File: styles/google-sign-in.scss (hunk 1 above)
Comment (maguero) [reply styles/google-sign-in.scss:6]: it shouldn't be used, I just copied from original google button. I tested without it and it seems to be OK, so I'm removing it.
Thanks

Comment (maguero) [reply styles/google-sign-in.scss:6]: I just copied from original google button. I tested without it and it seems to be OK, so I'm removing it.
Thanks

**Pull request merged**